src/l3-resources/atlas-bootstrap/index.ts

```diff
@@ -0,0 +1,129 @@
+import {
+  CfnOutput,
+  aws_iam as iam,
+  aws_secretsmanager as secretsmanager,
+  aws_cloudformation as cloudformation,
+  SecretValue,
+  Duration,
+} from "aws-cdk-lib";
+import { Construct } from "constructs";
+
+export const AtlasBasicResources: string[] = [
+  "Cluster",
+  "Project",
+  "DatabaseUser",
+  "ProjectIpAccessList",
+];
+
+export class MongoAtlasBootstrapProps {
+  /**
+   * The IAM role name for CloudFormation Extension Execution.
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry-public.html
+   * @default auto generat the name.
+   */
+  readonly roleName!: string;
+  /**
+   * The secret profile name for MongoDB Atlas.
+   * @default generate a dummy secret.
+   * @see https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master#2-configure-your-profile
+   */
+  readonly secretProfile!: string;
+  /**
+   * List of strings representing mongoDB atlas types to activate.
+   * the exported AtlasBasicResources can be used for AtlasBasic resources
+   * if not provided non resources will be activated
```

Comment:
Is there any use case that user would not activate any atlas basic resources?

Maybe when it's undefined, the default behavior is to activate all the basic resources instead? wdyt?

Reply:
mybe they have already activated them, or they want to manage them manually, so i left the option not to activate anything, or mybe choose witch resource they want to activate, mybe in the future they might want to modify the stack to add new ones

Reply:
also i see there is a few regions that does not have the AWS::CloudFormation::TypeActivation  me-central-1, ap-south-2, ap-southeast-4, eu-south-2, eu-central-2, so i just let the users decide not to activate them y the script, but do it manually